.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+scala:
+- 2.11.8
+- 2.11.11
+notifications:
+  slack:
+    secure: QcJzLxkSVl91qQnHD/DalJN2QuXcb2ac8V/mCb8TkzbB0fVX15o1M9JV8JHnIT0asjiNxrzne5CP/7jfnI3coQIuVkOlBV0gViLI1UO74G0wXij6Z2s+XhCaLfPiwlrE22NoipOtAC6aLMZ0EYbWw6vMCqQuXTi7QDqQKpCOgPks4/wbwi07AvxiBDBPBZXiciw7dkT0WDBM+XxhjvINTvgT9CuaQwSnLLBRwu0JjhVeaBTssx0kB85FyGhyrDq4N/36TsdJJRfGVOH20WXPQ8jqw0Vmnt3TkYaw8QTdIE1s2owio0xmrMkKpaWp/PJ1RQBFHLg7NmgjwOUby0YdYeg11OwhPIx7YfcPHrJon5QNPYc00ti7h4HEg3+vX/uPbi9P7RC/gYYM39p4OhNmfzs9azEMIKHRvz0m7r4QMKgTwUNdpLVm4+BgHk0Y2skNy+Nv5W6bH4Y5FbV/DjiK72XBq5N0s7mH2bM3bhOorUc4keXL5SxNCqbNjW+0cRiMjOxk1r63X7jlxO9UPQMBth6VZ2l+gCz/8iw/CPhXo1Qmzc0P5a8RD+sQVIbKDfrbnufC1AF659CSefrGRRKmUsnhjHexpA6/a5kCjGPYGbt6fgK/BwIi+eRjWP2q5IS1YZhJjG1hxtFXlrMU8WcPkHof42q34dhOgTBMIOsPc6I=
```

Comment:
It says "secure" because it's encrypted with a key connected to our specific repo.